cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -233,7 +233,7 @@ func ReadStaticPodFromDisk(manifestPath string) (*v1.Pod, error) {
 		return &v1.Pod{}, errors.Wrapf(err, "failed to read manifest for %q", manifestPath)
 	}
 
-	obj, err := kubeadmutil.UnmarshalFromYaml(buf, v1.SchemeGroupVersion)
+	obj, err := kubeadmutil.UnmarshalFromYaml(buf)
 	if err != nil {
 		return &v1.Pod{}, errors.Errorf("failed to unmarshal manifest for %q from YAML: %v", manifestPath, err)
```

Comment:
```suggestion
		return &v1.Pod{}, errors.Errorf("failed to unmarshal manifest for %q: %v", manifestPath, err)
```


NOTE: please remove any instances of YAML related to this unmarshaling or reading a file from disk with the universal decoder. the reason is it now should also work with JSON.

Reply:
Only used in func `ReadStaticPodFromDisk` and `PatchStaticPod`.

Updated.